src/evict/evict_lru.c

```diff
@@ -1273,8 +1262,7 @@ __evict_walk_file(WT_SESSION_IMPL *session, u_int *slotp, uint32_t flags)
 		 * running last time we wrote the page has since rolled back,
 		 * or we can help get the checkpoint completed sooner.
 		 */
-		if (modified && !LF_ISSET(
-		    WT_EVICT_PASS_AGGRESSIVE | WT_EVICT_PASS_WOULD_BLOCK) &&
+		if (modified &&
 		    (mod->disk_snap_min == S2C(session)->txn_global.oldest_id ||
```

Comment:
This part of the change should be unrelated.  Basically, this check is an optimization -- don't try to evict pages if we can tell with these quick checks that eviction will fail.